env.md

```diff
@@ -1,11 +1,31 @@
-Environment
-===========
+Supported Environments
+======================
+
+## Supported Cloud Providers
+
+* AWS (Amazon Web Services)
+  * All regions supported, as shown in [opencost/pkg/cloud/awsprovider.go​](https://github.com/opencost/opencost/blob/0c2f063052723a65ca62a4c75be23392806b6fac/pkg/cloud/awsprovider.go#L111)
+  * x86, ARM
+* GCP (Google Cloud Platform)
+  * All regions supported, as shown in [opencost/pkg/cloud/gcpprovider.go​](https://github.com/opencost/opencost/blob/0c2f063052723a65ca62a4c75be23392806b6fac/pkg/cloud/gcpprovider.go#L41)
+  * x86
+* Azure (Microsoft)
+  * All regions supported, as shown in [opencost/pkg/cloud/azureprovider.go​](https://github.com/opencost/opencost/blob/0c2f063052723a65ca62a4c75be23392806b6fac/pkg/cloud/azureprovider.go#L82)
+  * x86
+
+## Supported Cluster Types
+
+* ​Managed Kubernetes clusters (e.g. EKS, GKE, AKS) *most common*
+* ​Kubernetes distributions (e.g. OpenShift, DigitalOcean, Rancher, Tanzu)
+* ​Bootstrapped Kubernetes cluster​
+* On-prem and air-gapped using custom [pricing sheets](https://guide.kubecost.com/hc/en-us/articles/4407601795863#q-how-do-i-configure-prices-for-my-on-premise-assets)
+
+**This list is certainly not exhaustive!** This is simply a list of observations as to where our users run Kubecost based on their questions/feedback.
+
+## Supported Kubernetes Versions
 
 * Kubecost runs in any Kubernetes v1.8 (Aug 2017) or greater environment.
```

Comment:
Side note: are we ready bump this soon? This is really old these days...

Reply:
I agree that v1.8 is pretty dated ...

However! Explicitly stating which version Kubecost can run on may get ahead of any user questions!

Reply:
Are we confident it still runs on older versions? Do we want to support it on older versions?

Reply:
I personally, am only moderately confident because I haven't run these backwards compatibility tests myself.

Do we know of a team running these backward compatibility tests? If not, which Kubernetes versions are we comfortable saying that we support?

Reply:
For ent. users definitely want to support all the way back. I'm sure it will run, though I'm guessing not all features will be available.